README.md

```
@@ -16,7 +16,7 @@ Deployments in production:
  * http://vie.elimu.ai - Vietnamese database content
  * http://eng.elimu.ai - English database content
 
-[<kbd>![](https://github.com/elimu-ai/webapp/assets/15718174/32f3c339-aacc-4dc1-9692-c9435bc63d57)</kbd>](https://hin.elimu.ai)
+[<kbd>![](https://github.com/elimu-ai/webapp/assets/15718174/32f3c339-aacc-4dc1-9692-c9435bc63d57)</kbd>](http://hin.elimu.ai)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)